src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -102,6 +97,26 @@ $table-cell-height-half: $table-cell-height / 2;
   width: 15 * $baseline-grid;
 }
 
+.kd-replicasetdetail-tab-header {
+  md-tabs-canvas,
+  md-pagination-wrapper {
+    height: $toolbar-height-size-base;
```

Comment:
Hmm... Can you revert this 66px change from this PR. This does not look good to me, because the tabs are misaligned now.

Basically, I'm opposed to everything that is not standard Angular Material, unless it brings real benefits. The reason is that we have limited engineering resources and we cannot be implementing (and maintaining) our own UI components. 

What do you think?

@maciaszczykm